docs(www): Update npm run serve description (#11993)
gatsbyjs/gatsby: docs/docs/quick-start.md

npm run serve
```

> Gatsby starts a local HTML server for testing your built site. You must use `npm run build` before using this command.

Reviewer: ```suggestion
Gatsby starts a local HTML server for testing your built site. Remember to build your site using  `npm run build` before using this command.
```

Author: @sidharthachatterjee I think your suggestion reads nicer 👍

Reviewer: @andrecalvo I think you forgot to hit commit suggestion 😂

Author: @sidharthachatterjee oops! Apologies

Reviewer: Haha, no worries. Thank you for the contribution! Merging.
